auto_nag/team_managers.py

```diff
@@ -36,3 +40,81 @@ def get_team_manager(
                 return None
 
         return self.managers[team_name]
+
+    def _fetch_component_teams(self):
+        include_fields = [
+            "components.name",
+            "components.team_name",
+        ]
+
+        def handler(product, data):
+            data.update(
+                {
+                    component["name"]: component["team_name"]
+                    for component in product["components"]
+                }
+            )
+
+        # This is workaround until merging https://github.com/mozilla/libmozdata/pull/198
+        search = "type=accessible&include_fields=" + ",".join(include_fields)
+        BugzillaProduct(
+            search_strings=search,
+            product_handler=handler,
+            product_data=self.component_teams,
+        ).wait()
+
+    def _fetch_managers_nicnames(self):
+        people = People.get_instance()
+
+        bz_emails_map = defaultdict(list)
+        for manager in self.managers.values():
+            if not manager["mozilla_email"]:
+                continue
+
+            info = people.get_info(manager["mozilla_email"])
+            bz_mail = info["bugzillaEmail"]
+            if not bz_mail:
+                bz_mail = info["mail"]
+
+            # Some manager could manage multiple teams, we need to update all of
+            # them.
+            bz_emails_map[bz_mail].append(manager)
+
+        include_fields = [
+            "email",
+            "nick",
+        ]
+
+        bz_emails = [email for email in bz_emails_map.keys() if email]
+
+        def handler(user, data):
+            for manager in data[user["email"]]:
```

Comment:
Same here, you can avoid passing `data` and just use `bz_emails_map`

Reply:
Same as https://github.com/mozilla/relman-auto-nag/pull/1499#discussion_r900046603